tests/su3_test.cpp

```diff
@@ -72,41 +59,40 @@ void display_test_info()
 
 void add_su3_option_group(std::shared_ptr<QUDAApp> quda_app)
 {
-  CLI::TransformPairs<QudaWFlowType> wflow_type_map {{"wilson", QUDA_WFLOW_TYPE_WILSON},
-                                                     {"symanzik", QUDA_WFLOW_TYPE_SYMANZIK}};
+  CLI::TransformPairs<QudaGaugeSmearType> gauge_smear_type_map {{"ape", QUDA_GAUGE_SMEAR_APE},
+                                                                {"stout", QUDA_GAUGE_SMEAR_STOUT},
+                                                                {"ovrimp_stout", QUDA_GAUGE_SMEAR_OVRIMP_STOUT},
```

Comment:
for consistency with the rest of the CLI commands, change the underscore to a hyphen

Reply:
8c6db1f5c705446dfb5fed55428c694edc41e73e